### PR TITLE
[#113] Third party components list

### PR DESCRIPTION
**Pull request by pylapp**
To be more compliant with FLOSS principles, and also thanks developers of third-party components used in this project, the list of such components can be made in the app level and the repository level.

- [x] Add a folder "licenses" listing licenses of components used in the project
- [x] Add in Settings.bundle of the app a generated lit of components in use

### CODE REVIEW

**Comment by pylapp** [comment]
@victor-sarda @rozierguillaume Merge request related to issue #113 is ready 👍 

Available for review!

**Comment by pylapp** [comment]
@victor-sarda I reset the Settings.bundle and add the entry of the tool used to build the list of licenses.
Feel free to read this small contribution :)

**Comment by pylapp** [comment]
@victor-sarda From my point of view, no particular issue with this pull request.
Only a Settings.bundle is created, and it makes the app more compliant with FLOSS guidelines :)

**Comment by pylapp** [comment]
@victor-sarda About the merge request conflict, how do you want to resolve it? The _pbxproj_ is modified on my side because I created a settings bundle.
Do you want I update it on my side or should I let you update it with your evolutions? .pbxproj files are such a chore to fix when there are conflicts :(

**Comment by pylapp** [comment]
@victor-sarda Merge request ready again, fixed successfully conflict on pbxproj

**Comment by victor-sarda** [comment]
> @victor-sarda Merge request ready again, fixed successfully conflict on pbxproj 👍

@pylapp any chance you can provide screenshots? Thanks!

**Comment by pylapp** [comment]
For sure! You can find below three levels.
First, the app setting screen with the entry of the licenses list, and also a copyright (I thought it might be nice).
Then, the list of dependencies the tool has found directly and by transitions.
Finally, a sample of the result for a component.

In a nutshell: simple, clear, efficient 😄 

<table>
<tr>
<td>

</td>
<td>

</td>
<td>

</td>
</tr>
</table>

File: ViteMaDose.xcodeproj/project.pbxproj [hunk 1]
```diff
@@ -1181,17 +1187,18 @@
...
-				DEVELOPMENT_TEAM = ZL9396WK65;
+				DEVELOPMENT_TEAM = "";
 				INFOPLIST_FILE = ViteMaDose/Info.plist;
 				LD_RUNPATH_SEARCH_PATHS = (
 					"$(inherited)",
 					"@executable_path/Frameworks",
 				);
 				MARKETING_VERSION = 1.0;
-				PRODUCT_BUNDLE_IDENTIFIER = app.vitemadose;
+				PRODUCT_BUNDLE_IDENTIFIER = pylapp.app.vitemadose;
```
Comment (victor-sarda) [inline ViteMaDose.xcodeproj/project.pbxproj:1199]: Can you revert those changes please?

**Review by victor-sarda — CHANGES_REQUESTED** [review]
Nice one!
I just need you to revert the bundle id/signing in the `pbxproj` file 👍

**Comment by pylapp** [comment]
@victor-sarda That's done. Maybe you should update this file with your team however.

**Review by victor-sarda — APPROVED** [review]

**Pull request merged**